Applications/VGG/jni/meson.build

```diff
@@ -13,7 +15,7 @@ endif
 executable('nntrainer_vgg',
   mnist_sources,
   dependencies: [iniparser_dep, nntrainer_dep, nntrainer_ccapi_dep, app_utils_dep],
-  include_directories: include_directories('.'),
+  include_directories: [include_directories('.'), include_directories('../../utils/datagen/cifar')],
```

Comment:
Why don't we use  cifar_path?

Reply:
Meson build Error
-----------------
ERROR: Tried to form an absolute path to a source dir. You should not do that but use
relative paths instead.
-----------------
is there any solution to use absolute path ?

Reply:
i found below so i will fix this

```
cifar_include_path = join_paths(cifar_path, '/')
```

Reply:
How about declaring cifar_include_directories in utils/datagen/cifar/meson.build as follows
`cifar_include_directory = include_directories('.')`